SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -23,15 +23,12 @@ namespace SpatialGDK
 // It is built once per net driver initialisation.
 static QueryConstraint ClientCheckoutRadiusConstraint;
 
-InterestFactory::InterestFactory(AActor* InActor, const FClassInfo& InInfo, USpatialClassInfoManager* InClassInfoManager, USpatialPackageMapClient* InPackageMap)
-	: Actor(InActor)
-	, Info(InInfo)
-	, ClassInfoManager(InClassInfoManager)
-	, PackageMap(InPackageMap)
+InterestFactory::InterestFactory(USpatialClassInfoManager* InClassInfoManager)
+	: ClassInfoManager(InClassInfoManager)
 {
 }
 
-void InterestFactory::CreateClientCheckoutRadiusConstraint(USpatialClassInfoManager* ClassInfoManager)
+void InterestFactory::CreateClientCheckoutRadiusConstraint()
```

Comment:
Is this worth calling at initialisation now we have a set time when the interest factory should be "ready" (i.e. at initialisation)? Then the API would not include this function but offer the same external functionality, so you can't forget to call it